backend/scripts/monitor_pool.py

```
@@ -0,0 +1,106 @@
+            "checked_out": pool.checkedout(),
+            "overflow": pool.overflow(),
+            "invalid": pool.invalid(),
+            "total_connections": pool.size() + pool.overflow(),
+        }
+        
+        # Calculate utilization
+        max_connections = pool.size() + (pool._max_overflow if hasattr(pool, '_max_overflow') else pool.max_overflow if hasattr(pool, 'max_overflow') else 0)
+        stats["utilization_percent"] = round((stats["checked_out"] / max_connections * 100), 2) if max_connections > 0 else 0
+        stats["max_connections"] = max_connections
+        
+        return stats
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)